packages/app-builder-lib/src/node-module-collector/index.ts

```diff
@@ -0,0 +1,28 @@
+import { NpmNodeModulesCollector } from "./npmNodeModulesCollector"
+import { PnpmNodeModulesCollector } from "./pnpmNodeModulesCollector"
+import { YarnNodeModulesCollector } from "./yarnNodeModulesCollector"
+import { detect, PM, getPackageManagerVersion } from "./packageManager"
+import { NodeModuleInfo } from "./types"
+import { log } from "builder-util"
+
+async function getCollectorByPackageManager(rootDir: string) {
+  const manager: PM = await detect({ cwd: rootDir })
+  switch (manager) {
+    case "npm":
+      return new NpmNodeModulesCollector(rootDir)
+    case "pnpm":
+      return new PnpmNodeModulesCollector(rootDir)
+    case "yarn":
+      return new YarnNodeModulesCollector(rootDir)
+    default:
+      log.filePath(rootDir)
```

Comment:
Please remove. `log.filePath(...)` returns a string as well, so this line isn't doing anything.

Reply:
removed